scripts/cli-test.sh

```diff
@@ -6,7 +6,6 @@ export RUST_LOG=info
 export PEG_IN_AMOUNT=99999
 
 source ./scripts/lib.sh
```

Comment:
is the call to lib.sh redundant here?

Reply:
yes right. I did a small test and thought that this is not possible but my test was wrong